compendium/postchapters/build.tex

```diff
@@ -64,7 +64,7 @@ \subsection{Installera sbt}
 
 \item \textbf{Linux}. Om du surfar till ovan sida från en Linux-dator syns några terminalkommando som du använder för att installera \sbt\ i terminalen.
 
-\item \textbf{Windows}. Om du surfar till ovan sida från en Windows-dator visas en länk till en \code{.msi}-fil. Ladda ner och dubbelklicka på den.
+\item \textbf{Windows}. Om du surfar till ovan sida från en Windows-dator visas en länk till en \code{.msi}-fil. Ladda ner och dubbelklicka på den. Innan du börjar använda sbt i en Windowsterminal rekommenderas det att skriva \code{chpcp 65001} för att särskilda tecken (t.ex. ÅÄÖ) ska fungera som de ska.
```

Comment:
Suggestion: "börjar använda" -> "kör igång"  as 'start using' may sound as a one-off thing...

Reply:
And maybe also "rekommenderas det" -> "är det bra att skriva"  to make it active voice instead of passive voice.

Reply:
I will fix. Is it a good location in the compendium?

Reply:
And just found one error: "chpcp" -> "chcp"